Milestone 01: Setup enviroment & Parse Figma Template

s2jack opened the pull request:
This is the first PR of this project.

1- Create HTML with toolbar and headline
2- Create CSS file to organize text, shapes, buttons color, and padding.
3- Set up linters for HTML & CSS
4- Add .gitignore
5- Add Descriptive README.md file
6- Add images source folder

s2jack committed:
Create linters.yml

Add HTML&CSS linters file into project

s2jack committed:
Add index.html

Add 3 sections:
1-Navbar
2-Headline
3-Bot Separator

Add images
Add button
Add onhover effect to button
Add Headline text

s2jack committed:
Add styles.css

Set alignments
Set font size
Set colors

s2jack committed:
Add images folder

Add images source folder for buttons and shapes

s2jack committed:
Create README.md

Fill in project information

s2jack committed:
Fix Styleint linter error

Fix missing new line error on styles.css

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,42 @@
+        <div id="icmenu">
+          <img src="images/ic_menu.svg" width="40" height="28">
+        </div>
+      </nav>
```
- [x] You have reached a great milestone by have a logo. However the logo should be wrapped using [`<header>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) tag. The <header> element represents a container for introductory content or a set of navigational links which include but not limited to logo or title. Use the [ux stackexchange conversation](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) to have a clear understanding about this topic.

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,42 @@
+      <nav id="navbar">
+        <div id="logo">
+          <img src="images/image_geometry_1.svg" width="111" height="74">
+        </div>
```
- [x] If you notice most of the websites, when you click on the logo, you are directed to home page. This implies that the logo should be a link and hence `a` tag should be used instead of `<div>` tag. For [Semantic purposes](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwi56cjW38D8AhX0UaQEHVp7DLEQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU), kindly ensure that the logo image is a link to home page.

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,42 @@
+        </div>
+        <div id="icmenu">
+          <img src="images/ic_menu.svg" width="40" height="28">
+        </div>
```
- [x] I would suggest to wrap the `bar` icon in the `<nav>` tag :relaxed: . The hamburger menu is a space-saving mechanism since all navigation options become visible only after a click or tap. Therefore since the should contain different menu options, it could be better to have a `nav` tag for Semantic purposes and [user experience](https://developer.mozilla.org/en-US/docs/Glossary/UX) :sunglasses:

AlphaNtihinduka commented on styles.css:
```diff
@@ -0,0 +1,35 @@
+#navbar {
+  height: 13%;
```
- [x] Kindly consider the use of margins(top, left, right, or bottom) to implement the design :ballot_box_with_check: . You should also ensure that you stick to the original [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=cSMRQuqk7mevelEo-0). This makes the webpage more professional.
- Your design
![Screenshot from 2023-01-12 03-36-36](https://user-images.githubusercontent.com/99394374/211948355-b8409a34-23a6-412c-a5b6-5e5cd48796b7.png)

- Figma design
![Screenshot from 2023-01-12 03-37-41](https://user-images.githubusercontent.com/99394374/211948362-94a2f40b-e2e2-4ff8-a238-c91b9f4480a2.png)

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,42 @@
+        <div id="name">
+          Mehmet Selçuk
+          Güler
+        </div>
```
- [x] To enable google or other search engines to know the specific topic, you should consider improving the [SEO](https://developers.google.com/search/docs/fundamentals/seo-starter-guide) of the website. Therefore you should consider use of `<h1>` tag which is usually used for the title of a page or post and it is the first header visible on a page. Consider referring to this [article](https://www.reliablesoft.net/h1-tag/) for better understanding.

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,42 @@
+          feature or website. Taka a look of my works.
+          If you like what you see and have a project you need coded, 
+          don’t hesitate and contact me.
+        </div>
```
- [x] Kindly consider the use of `<p>` tag on paragraphs. :ok_hand:  This enables the screen readers like [NVDA](https://www.nvaccess.org) to have easy access to the webpage and also this is good for [Semantic reasons](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwih0amB6cD8AhVAnP0HHXTNDzYQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU)

AlphaNtihinduka commented on styles.css:
```diff
@@ -0,0 +1,35 @@
+  padding-top: 5em;
+}
+
+.headline {
+  margin-left: 2.4rem;
+  display: flex;
+  width: 83%;
+  height: 55%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#name {
+  padding-top: 5%;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 3.4em;
+}
+
+#message {
+  font-size: 1.5em;
+  font-family: Arial, Helvetica, sans-serif;
+}
```
- [x] If you notice, there are variations in color, font, width, and spacing to the figma design. Kindly ensure that the design is professionally following the figma design.
- Your design :sunglasses: 


![Screenshot from 2023-01-12 04-10-19](https://user-images.githubusercontent.com/99394374/211952038-fda25b4e-c437-41a0-a5dc-4e2f07065fcc.png)

- Figma design :sunglasses: 

![Screenshot from 2023-01-12 04-09-47](https://user-images.githubusercontent.com/99394374/211952086-90e23563-9558-46f9-aa07-2eb0fad3209b.png)

AlphaNtihinduka commented on index.html:
```diff
@@ -0,0 +1,42 @@
+          Mehmet Selçuk
+          Güler
+        </div>
+        <div id="message">
```
- [ ] [OPTIONAL] You should kindly prefer using `class` selectors instead of `id`. This is because  **a class can be used to identify more than one HTML element.** :ok_hand:

AlphaNtihinduka reviewed (changes requested):
## Hi @s2jack     🙋‍♂️
Good job working on your first mobile setup. You have done the design to the best expectation and incredibly well. 👏  
However, there are some issues mostly on the `HTML tags` for [semantic HTML](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjwyP2CrLD8AhUZi_0HHdAIB8UQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU) and font. 
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional :heavy_check_mark: 
- Professional README file :100: 
- Your PR is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

## Required changes :recycle: 
- Kindly consider the inline comments that clear describe each change required.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

s2jack committed:
Add Fonts folder

Add fonts source folder that contains all fonts in the index.html

s2jack committed:
Update index.html

Add a new tag <main> to organize all margins of main page elements.
Change tag to <header> of "toolbar"
Add a home page link to the "Logo"
Change tag to <nav> of "menu bar"
Change tag to <h1> of name title
Change tag to <p> of message text

s2jack committed:
Update style.css

Add fonts
Fix margins and paddings
Use class selector for text
Add text-shadow property
Fix differences between Figma template and index.html page

s2jack committed:
Update style.css

Add generic font family
Delete useless spaces and blank lines
Fix Stylint linter errors

s2jack committed:
Update styles.css

Fix space issues
